### PR TITLE
Shut these fucking messages up

### PR DESCRIPTION
**Pull request by jimcarreer**

### CODE REVIEW

File: slack/tasks.py [hunk 1]
```diff
@@ -29,8 +29,9 @@ def process_event(data):
     # noinspection PyBroadException
     try:
         if data['event'].get('subtype') in SUBTYPE_BLACKLIST:
-            logger.debug(f'Ignoring {data["event"]["event_id"]}, subtype was in blacklist')
-            return
+            return logger.debug(f'Ignoring {data["event"]["event_id"]}, subtype was in blacklist')
```
Comment (zachtib) [inline slack/tasks.py]: I've gotten KeyErrors on event_id before, just fyi

File: slack/test_data.py [hunk 2]
```diff
@@ -142,3 +142,49 @@
...
+        "subtype": "message_deleted",
+        "ts": "",
+        "type": "message"
+    },
+    "event_id": "EvUB8VV90V",
+    "event_time": 1582297852,
+    "team_id": "T5J4V03V4",
+    "token": "SUCKATHISISSECRET",
+    "type": "event_callback"
+}'''
```
Comment (zachtib) [inline slack/test_data.py]: Newline

File: slack/tasks.py (hunk 1 above)
Comment (jimcarreer) [reply slack/tasks.py]: Ill just call get on the dict, its a log anyway.

**Comment by jimcarreer** [comment]
Force pushed a rebase

**Pull request merged**